ci/Jenkinsfile.e2e

```diff
@@ -138,10 +139,6 @@ pipeline {
             squishPackageName: 'squish-6.7.2-qt514x-linux64',
             testSuite: "${WORKSPACE}/test/ui-test/testSuites/*",
           ])
-          print("Squish run result: ${result}")
-          if (!['SUCCESS', 'UNSTABLE'].contains(result)) {
```

Comment:
we need to keep this as there is a bug in squish/jenkins and retry, when retry succeed, the test are still mark as failed. This is why we also have this unstable

Reply:
This is why God invented code comments. Because this kind of knowledge disappears if not included in the code.

Thanks for explaining.

Reply:
So for example in this case:
https://ci.status.im/job/status-desktop/job/branches/job/e2e/job/PR-8386/1/

![image](https://user-images.githubusercontent.com/2212681/203571460-ded4d75b-a365-48fc-b1c0-689f97420edb.png)

There are tests that failed, but the result was `UNSTABLE`:
```
10:34:43  Squish run result: UNSTABLE
```
https://ci.status.im/job/status-desktop/job/branches/job/e2e/job/PR-8386/1/console

@alaibe is that the intended behavior because the tests were retried?

Reply:
Ok. Apparently `UNSTABLE` in that case is an issue with Squish itself.

I'm changing the PR to just add a comment explaining why we ignore `UNSTABLE`.